res/css/views/rooms/_AppsDrawer.pcss

```diff
@@ -325,6 +325,9 @@ limitations under the License.
     &.mx_AppTileBody--call {
         border-radius: 0px;
     }
+    &.mx_AppTileBody--call.mx_AppTileBody--mini {
+        border-radius: 8px;
+    }
```

Comment:
why is this now needed (as well as the border-radius on `.mx_WidgetPip_overlay ` ?

(I have no real idea what a `&.mx_AppTileBody--call.mx_AppTileBody--mini` is, I'm afraid)

Reply:
Before the widget was always rendered the default app container with rounded borders.
This is a concious design decision to underline that it is not part of Eement web but potentially a thrid party app running as a widget. For calls we want to special case this and show the widget as if its part of Element Web.
So `mx_AppTileBody--call` has `border-radius: 0px;`.
For the PiP call view rounded corners are alwasy required. So we set it back to 8px.

Old:
<img width="653" alt="Screenshot 2024-01-05 at 21 49 58" src="https://github.com/matrix-org/matrix-react-sdk/assets/16718859/dadfb367-9bc1-4a68-acf2-2ea19553adb2">
Now:
<img width="653" alt="Screenshot 2024-01-05 at 21 56 58" src="https://github.com/matrix-org/matrix-react-sdk/assets/16718859/dc82f04e-a1ff-4a16-b845-e4e3929cd9da">
Issue caused by new design
<img width="200" alt="Screenshot 2024-01-05 at 21 53 16" src="https://github.com/matrix-org/matrix-react-sdk/assets/16718859/036de163-49ef-4654-81c8-8ca84047d129">
Fix for new design:
<img width="200" alt="Screenshot 2024-01-05 at 21 53 49" src="https://github.com/matrix-org/matrix-react-sdk/assets/16718859/21f0eb13-ac33-40c6-b5cb-f6bf40a5a65c">

Reply:
> Before the widget was always rendered the default app container with rounded borders.

I don't follow. There don't seem to be enough words in this sentence?

> alwasy

always